app/services/address_finder_service.rb

```diff
@@ -20,7 +20,11 @@ def search_by_postcode
       :not_found
     rescue RestClient::ExceptionWithResponse => e
       Airbrake.notify(e)
-      Rails.logger.error "Os Places error: " + e.to_s
+      Rails.logger.error "OS Places error: " + e.to_s
```

Comment:
Do you think there is any value tweaking the message for these errors to differentiate them in the log e.g. `OS Places response error` and `OS Places socket error`?

Reply:
Sounds like a good idea to me. I'll tweak it now.

Reply:
@Cruikshanks Fixed!